playbooks/provisioning/openstack/provision-openstack.yml

```diff
@@ -8,6 +8,10 @@
     - include: pre_tasks.yml
   roles:
     - role: openstack-stack
+    - role: static_inventory
+      when: openstack_inventory|default('static') == 'static'
+      inventory_path: "{{ openstack_inventory_path|default('../../../../inventory') }}"
+      private_ssh_key: "{{ openstack_private_ssh_key|default('~/.ssh/openshift') }}"
```

Comment:
Because the README now removes the `--private-key` call from `ansible-playbook`, the default key should be `~/.ssh/id_rsa`.